### ISSUE TITLE
TypeError on Deno Deploy

### ISSUE THREAD

**Issue by hasundue** [upstash/upstash-redis#266]
Appreciate for supporting Deno, first of all 😃 

Just found out that the latest release has introduced an error on Deno Deploy:

```sh
TypeError: Cannot read properties of undefined (reading 'deno')
    at new Redis (https://deno.land/x/upstash_redis@v1.18.0/mod.ts:26:54)
    at file:///src/app/redis.ts:3:22
```

The error occurs at a typical constructor but the code is [here](https://github.com/hasundue/denopendabot/blob/main/app/redis.ts) for reference.

The problem is that the [Deno Deploy Runtime](https://deno.com/deploy/docs/runtime-api) does not have `Deno.version` API, which is used in the telemetry part:

```typescript
    if (!Deno.env.get("UPSTASH_DISABLE_TELEMETRY")) {
      telemetry.runtime = `deno@${Deno.version.deno}`;
      telemetry.sdk = `@upstash/redis@${VERSION}`;
    }
```

I could create a PR to fix this but would like to know what policy I should follow to fix it in that case.

**Comment by chronark** [comment]
Oh I see, thank you.
Do you know if there is a different way to access the version in demo deploy?

**Comment by SagarMahadik** [comment]
import { Redis } from "https://deno.land/x/upstash_redis@v1.16.0/mod.ts"; @chronark  - This version worked for me.

**Comment by chronark** [comment]
Fixed it in `v1.18.1`
Thank you
